song-server/src/main/resources/db/migration/V1_16__specimen_type_update_enum.sql

```diff
@@ -0,0 +1 @@
+ALTER TYPE specimen_type RENAME VALUE 'Malignant neoplasm without specification of site' TO 'Tumour - unknown if derived from primary or metastatic'
```

Comment:
There is already a V16 script exists in song, please rename it to V17.

Reply:
Filename changed.